eng/Version.Details.xml

```diff
@@ -98,22 +98,22 @@
       <Sha>0385265f4d0b6413d64aea0223172366a9b9858c</Sha>
       <SourceBuild RepoName="test-templates" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="7.0.100-rc.1.22409.4">
+    <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="7.0.100-dev">
```

Comment:
we are back to -dev versions..

Reply:
@vlada-shubina merged SDK PR https://github.com/dotnet/sdk/pull/27083 that should address this - it just needs to first go through _dotnet-sdk-official-ci_